files/ru/glossary/abstraction/index.md

```diff
@@ -41,9 +41,3 @@ class ImplementAbstraction {
   // a = 10
   // b = 20
 ```
-
-## Дополнительные ресурсы
-
-### Общие сведения
-
-- {{interwiki("wikipedia", "Абстракция (информатика)", "Абстракция")}} на Википедии
```

Comment:
Hi! I believe as per this comment https://github.com/orgs/mdn/discussions/165#discussioncomment-3296848 we shouldn't delete macro but convert it into md-link pointing to corresponding wikipedia-page

Reply:
Yes, I converted part of them, which in my opinion is important. Links from glossary I marked as non-important, because our glossary should give the explanation instead of Wiki. So, it's more important to improve the Glossary.